multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/parser/StagingParametersParser.java

```diff
@@ -57,6 +69,10 @@ public Staging parse(List<Map<String, Object>> parametersList) {
                                .invocationTimeout(healthCheckInvocationTimeout)
                                .healthCheckType(healthCheckType)
                                .healthCheckHttpEndpoint(healthCheckHttpEndpoint)
+                               .readinessHealthCheckType(readinessHealthCheckType)
+                               .readinessHealthCheckHttpEndpoint(readinessHealthCheckHttpEndpoint)
+                               .readinessHealthCheckTimeout(readinessHealthCheckInvocationTimeout)
```

Comment:
maybe it is good to align names to match `readinessHealthCheckInvocationTimeout`